File: src/main/java/com/kevin/guess/Game.java
package com.kevin.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secret = random.nextInt(10) + 1;

        Scanner scanner = new Scanner(System.in);
        int number = 0;

        while (true) {
            System.out.print("Please enter a number: ");
            number = scanner.nextInt();
            System.out.println((number>secret)?"Lower":"Higher");
            if (number == secret) {
                System.out.println("Good guess! Thank you for playing.");
                break;
            }
        }

//        do {
//            System.out.print("Please enter a number: ");
//            number = scanner.nextInt();
//            System.out.println(number);
//        } while (number != 0);
//        }
    }
}

File: src/main/kotlin/com/kevin/Game.kt
package com.kevin

import java.util.Scanner

fun main() {
    val secret = (1..10).random()
    println(secret)
    var number = 0
    Scanner(System.`in`).use { scanner ->
        while (number != secret) {
            print("Please enter a number: ")
//            number = scanner.nextInt()
            number = readln().toInt()
            if (number > secret) {
                println("Lower")
            } else if (number < secret) {
                println("Higher")
            } else {
                println("Great, the number is $number")
            }
        }
    }
}